actions/vulnerability_scan/action.yaml

```diff
@@ -0,0 +1,50 @@
+name: Vulnerability Scanning
+description: Scans for vulnerabilities
+inputs:
+  scan_severity:
+    description: The severity that will cause the action to report if a vulnerability at that level is detected. UNKNOWN,LOW,MEDIUM,HIGH,CRITICAL
+    default: CRITICAL,HIGH
+  failure_severity:
+    description: The severity that will cause the action to fail if a vulnerability at that level is detected. UNKNOWN,LOW,MEDIUM,HIGH,CRITICAL
+    default: CRITICAL
+  publish_vulnerabilities:
+    description: If true, will attempt to publish the results to the GitHub security tab
+    default: true
+  image_ref: 
+    description: The image reference to scan vulnerability on. If set it will be a image scan.
+    default: ''
+  scan_type:
+    description: The scan-type for aquasecurity/trivy-action action. Default to a fs scan.
+    default: fs
+runs:
+  using: "composite"
+  steps:
+  - name: Generate Trivy vulnerability scan report
+    uses: aquasecurity/trivy-action@0.14.0
+    if: inputs.publish_vulnerabilities == 'true'
+    with:
+      image-ref: ${{ inputs.image_ref }}

```

Comment:
This value is only needed when `scan-type` is `image`. So default it to empty: https://github.com/aquasecurity/trivy-action